### PR TITLE
new definition: multisort package

### PR DESCRIPTION
**Pull request by CzBuCHi**
If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.

https://github.com/peterkhayes/multisort

### CODE REVIEW

**Comment by CzBuCHi** [comment]
wrong inport signature

**Comment by CzBuCHi** [comment]
fixed syntax

File: types/multisort/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": false,
```
Comment (mhegazy) [inline types/multisort/tsconfig.json]: this should be true for a new package.

File: types/multisort/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        "strictNullChecks": false,
```
Comment (mhegazy) [inline types/multisort/tsconfig.json]: and same here.

File: types/multisort/tslint.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```
Comment (mhegazy) [inline types/multisort/tslint.json]: for new packages we would like them to lint clean. so no exclusions.

**Review by mhegazy — CHANGES_REQUESTED** [review]

**Pull request merged**